modules/test/playwright/tests/commerce/commerceChannelDefaults.spec.ts

```diff
@@ -200,7 +200,7 @@ test.describe('LPD-26142 Refactor of CommerceSalesAgent#CanSalesAgentWithPermiss
 			siteGroupId: site.id,
 		});
 
-		await performLogout(page);
+		await performLogout(page, 'test');
```

Comment:
Avoid refactoring all performLogout uses, I have some ideas to discuss

Reply:
I made this consistent with the `LoginScreenName.performLogin` method.

I was tempted to add a default value when i did it (something similar to:

```
async goto(siteUrl?: Site['friendlyUrlPath']) {
		await this.page.goto(
			`/group${siteUrl || '/guest'}${PORTLET_URLS.messageBoardsAdmin}`
		);
	}
```
But I decided to do it simple.

Anyway, if you have any suggestions, please tell me